### DISCUSSION TITLE
首先感谢分享，请问预训练模型方便告知一下吗？或者说能分享下从零开始训练的方法。 另外能适配下mps吗

### DISCUSSION THREAD

**Discussion by z20150214** [OleehyO/TexTeller#46] [Q&A] [answered]

**Comment by OleehyO** [comment, marked as answer]
TexTeller没有使用预训练模型，模型架构类似于[TrOCR](https://arxiv.org/pdf/2109.10282.pdf)，也可以看一下[TexTeller.py](https://github.com/OleehyO/TexTeller/blob/main/src/models/ocr_model/model/TexTeller.py)。

训练方法就写在[train.py](https://github.com/OleehyO/TexTeller/blob/main/src/models/ocr_model/train/train.py)里，其实就是端到端训练，没有什么特别的。

mps适配的话之后会考虑加进去。

**Comment by z20150214** [comment]
感谢解答，昨天没仔细看代码。
